Podfile

```diff
@@ -52,7 +52,6 @@ target 'AzureCommunicationChat' do
     inherit! :search_paths
     pod 'OHHTTPStubs/Swift'
     pod 'Trouter', '0.0.1-beta.5'
-    pod 'MSAL', '1.1.15'
```

Comment:
What is this pod and there a reason why it is being removed?

Reply:
It's not used by Chat so there's no reason for it to be listed as a dependency.